packages/interface/src/components/explorer/FileColumns.tsx

```diff
@@ -0,0 +1,42 @@
+export interface IColumn {
+	column: string;
+	key: string;
+	width: number;
+}
+
+export const LIST_VIEW_HEADER_HEIGHT = 40;
+
+// Function ensure no types are lost, but guarantees that they are Column[]
+export function ensureIsColumns<T extends IColumn[]>(data: T) {
+	return data;
+}
+
+export const columns = ensureIsColumns([
+	{ column: 'Name', key: 'name', width: 280 } as const,
+	// { column: 'Size', key: 'size_in_bytes', width: 120 } as const,
+	{ column: 'Type', key: 'extension', width: 150 } as const,
+	{ column: 'Size', key: 'size', width: 100 } as const,
+	{ column: 'Date Created', key: 'date_created', width: 150 } as const,
+	{ column: 'Content ID', key: 'cas_id', width: 150 } as const
+]);
```

Comment:
`ensureIsCoumns` is redundant - we have `satisfies`
```suggestion
export const columns = [
	{ column: 'Name', key: 'name', width: 280 } as const,
	// { column: 'Size', key: 'size_in_bytes', width: 120 } as const,
	{ column: 'Type', key: 'extension', width: 150 } as const,
	{ column: 'Size', key: 'size', width: 100 } as const,
	{ column: 'Date Created', key: 'date_created', width: 150 } as const,
	{ column: 'Content ID', key: 'cas_id', width: 150 } as const
] as const satisfies IColumn[];
```

Reply:
@Brendonovich the suggested change didn't work here, IColumn has a type error because its "readonly".

Reply:
Ah, use `satisfies Readonly<IColumn[]>` instead